spec/features/comments_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+include AuthHelper
+
+RSpec.describe "Article comments", type: :feature do
+  describe "comments form" do
+    let!(:article) { create(:article) }
+
+    it "should show user comment form on page" do
+      visit article_path(article)
+
+      expect(page).to have_content "Name"
```

Comment:
I would say this example is already covered int he example below, so isn't needed